service/matching/handler.go

```diff
@@ -59,7 +61,7 @@ type (
 		logger            log.Logger
 		startWG           sync.WaitGroup
 		throttledLogger   log.Logger
-		namespaceRegistry namespace.Registry
```

Comment:
The vast majority of changes look like this. Other bits were moved are called in identical fasion.  MockRegistry, NewRegistry, and NewMockRegistry